mlir/utils/jenkins/Jenkinsfile

```diff
@@ -256,7 +263,14 @@ pipeline {
                             }}
                             steps {
                                 dir('MIOpen') {
-                                getAndBuildMIOpen('--prefix ./MIOpenDeps', '''-DMIOPEN_USE_MLIR=On
+                                git branch: 'develop', poll: false,\
+                                    url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
+                                sh 'sed -i "/ROCmSoftwarePlatform\\/llvm-project-mlir/d" ./requirements.txt'
+                                cmake arguments: "-P install_deps.cmake --minimum --prefix ${WORKSPACE}/MIOpen/MIOpenDeps",\
+                                    installation: "InSearchPath"
+                                sh "cp ${WORKSPACE}/libMLIRMIOpen/lib/libMLIRMIOpen.a ${WORKSPACE}/MIOpen/MIOpenDeps/lib"
+                                sh "cp ${WORKSPACE}/libMLIRMIOpen/include/Miir.h ${WORKSPACE}/MIOpen/MIOpenDeps/include"
```

Comment:
No, we should let `igemm build` stage to handle the installation not to manually do it like this.